controllers/util/util.go

```diff
@@ -103,3 +105,21 @@ func AssertEqual[T comparable](actual T, expected T, exitCode int) {
 		os.Exit(exitCode)
 	}
 }
+
+// IsAnnotationTruthy returns true if the annotation is present
+// and has a truthy value
+func IsAnnotationTruthy(obj metav1.Object, key string) bool {
+	annotations := obj.GetAnnotations()
+
+	if val, found := annotations[key]; found {
+		return strings.ToLower(val) == "true"
+	}
+	return false
```

Comment:
The entire function just equal to
```
return obj.GetAnnotations()(key) == "true"
```

And it that case we don't need a util at all, just use the above line in the call sites